Formula/rtptools.rb

```diff
@@ -0,0 +1,31 @@
+class Rtptools < Formula
+  desc "Set of tools for processing RTP data"
+  homepage "https://www.cs.columbia.edu/irt/software/rtptools/"
+  url "https://www.cs.columbia.edu/irt/software/rtptools/download/rtptools-1.21.tar.gz"
+  sha256 "2ddbf6d3a4dbdef26ca85e860f6795264a359d30e24fd63f14a8c4f7ca331830"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    packet = [
+      0x5a, 0xb1, 0x49, 0x21, 0x00, 0x0d, 0x21, 0xce, 0x7f, 0x00, 0x00, 0x01,
+      0x11, 0xd9, 0x00, 0x00, 0x00, 0x18, 0x00, 0x10, 0x00, 0x00, 0x06, 0x8a,
+      0x80, 0x00, 0xdd, 0x51, 0x32, 0xf1, 0xab, 0xb4, 0xdb, 0x24, 0x9b, 0x07,
+      0x64, 0x4f, 0xda, 0x56
+    ]
+
+    File.open(testpath/"test.rtp", "wb") do |file|
```

Comment:
can't use a heredoc?

Reply:
I used `File.open` because of the combination of the first line being an Ascii string, and the rest of the data being binary.

Reply:
Let's use
```
    (testpath/"test4.rtp").open("wb") do |f|
      f.puts "#!rtpplay1.0 127.0.0.1/55568"
      f.write packet.pack("c*")
    end
```

Reply:
Ok, change made